doc/user_guide/recordflux_ug/03-getting_started_with_recordflux.rst

```diff
@@ -0,0 +1,726 @@
+.. _Getting_Started_With_RecordFlux:
+
+Getting Started with RecordFlux
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This chapter introducess the basic features of the RecordFlux toolset
+through a simple but representative example.
+
+.. _What_Is_RecordFlux:
+
+What Is RecordFlux?
+===================
+
+.. index Domain-specific language (DSL)
+
+RecordFlux comprises a domain-specific language (DSL) for precisely
+describing complex binary data formats and communication protocols,
+togther with a toolset to formally verify specifications, generate
+provable code in the SPARK subset of the Ada language, and validate
+specifications using communication traces and existing implementations.
+
+The formal specification in RecordFlux's DSL serves as a single source
+of truth in this process. Due to its abstract nature and its
+optimization of binary formats and communication protocols, a
+RecordFlux specification can be written and understood by domain
+experts who are not necessarily programmers or verification specialists.
+The SPARK code that is generated from a valid RecordFlux specification
+can automatically be proven to be free of runtime errors (thus no buffer
+overrun or integer overflow), and key properties -- namely the behavior
+with respect to the specification -- can be shown.
```

Comment:
I think the RecordFlux overview image, which shows all parts of the RecordFlux toolset, could fit in well here. @senier could you please provide a recent version of this image?

Reply:
I shared the current version with you.

Reply:
![RecordFlux Architecture](https://user-images.githubusercontent.com/15232394/204809702-915202b7-36d1-48a0-8657-c27be6ad3cfd.svg)